src/scripts/components/widgets/modals/Modal.js

```diff
@@ -65,23 +75,57 @@ const SVG = styled.svg`
 
 const Content = styled.div`
   height: 100%;
-  padding: 40px 46px;
+  padding: 44px 48px;
+  width: 100%;
+`
+
+export const ModalContentWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
   width: 100%;
 `
 
+export const ModalScrollContent = styled.div`
+  flex: 1 1 100%;
+
+  @media (max-width: 767px) {
+    overflow-x: hidden;
+    overflow-y: scroll;
+  }
+`
+
 class Modal extends Component<Props, void> {
+  componentDidMount () {
+    document.addEventListener('keydown', event => {
+      if (event.keyCode === 27) {
+        this.props.closeModal()
+      }
+    })
```

Comment:
Looks much better. One more thing that should be done, as with the current way this is written we will add new listeners every time we mount the component and they will never be removed.

- move the callback into a method on this component
- then, define `componentWillUnmount`

```javascript
handleKeydown = (event) => {
  if (event.keyCode === 27) {
    this.props.closeModal()
  }
}

componentDidMount() {
  document.addEventListener(this.onKeydown);
}

componentDidUnmount() {
  document.removeEventListener('keydown', this.onKeydown);
}
```

Reply:
Done https://github.com/Paratii-Video/paratii-portal/pull/301/commits/1afdcbafae637b40166bddfec94ae502b784ccbd